stan/math/prim/mat/fun/inv_Phi.hpp

```diff
@@ -0,0 +1,44 @@
+#ifndef STAN_MATH_PRIM_MAT_FUN_INV_PHI_HPP
+#define STAN_MATH_PRIM_MAT_FUN_INV_PHI_HPP
+
+#include <stan/math/prim/mat/vectorize/apply_scalar_unary.hpp>
+#include <stan/math/prim/scal/fun/inv_Phi.hpp>
+#include <stan/math/prim/scal/err/check_bounded.hpp>
+
+namespace stan {
+  namespace math {
+
+    /**
+     * Structure to wrap inv_Phi() so it can be vectorized.
+     * @param x Variable in range [0, 1].
+     * @tparam T Variable type.
+     * @return Inverse unit normal CDF of x. 
+     * @throw std::domain_error if x is not between 0 and 1.
+     */
+    struct inv_Phi_fun {
+      template <typename T>
+      static inline T fun(const T& x) {
+        using stan::math::inv_Phi;
+        stan::math::check_bounded<T, T, T>(
+          "inv_Phi vectorize", "Probability variable", x, 0, 1);
```

Comment:
The types here are incorrect for `check_bounded`, but I think we should be able to simplify this:
```
stan::math::check_bounded("inv_Phi", "Probability variable", x, 0.0, 1.0);
```
The types will be inferred properly here and we shouldn't promote 0 and 1 to `T`.
Note: I'm also changing the function name from "inv_Phi vectorize" to "inv_Phi" to match what the user will see.